UPGRADING.md

```diff
@@ -2,6 +2,10 @@
 
 This document describes breaking changes and how to upgrade. For a complete list of changes including minor and patch releases, please refer to the [changelog](CHANGELOG.md).
 
+## v5
+
+Upgraded `abstract-leveldown` to `v6.0.0`. Please see corresponding [changelog entry](https://github.com/Level/abstract-leveldown/blob/master/CHANGELOG.md#600---2018-10-20) for more information.
```

Comment:
```suggestion
 Upgraded `abstract-leveldown` to `v6.0.0`. Please see the corresponding [changelog entry](https://github.com/Level/abstract-leveldown/blob/master/CHANGELOG.md#600---2018-10-20) for more information.
```

And to make this referral less of a rabbit hole, I think we should reorder the `abstract-leveldown` changelog a bit. First describe changes in the public API (serialization etc, which is inherited by every implementation), then details which only matter to implementors (tests etc). I'll open an issue.